### DISCUSSION TITLE
CryptoBank Africa

### DISCUSSION THREAD

**Discussion by cryptobanks-app** [filecoin-project/community#239] [Project Showcase]
CryptoBank aims to provide a seamless integration that gives e-commerce businesses(merchants/private) the power to accept digital currencies-A new way to pay with digital currency with the interoperability characteristic that allows our users to make payments with any major digital currencies and tokens BTC, ERC-20, BEP-20, Filecoin, NUL-20, TRC-20 e.t.c. Our flexible solution allows our users to decide which digital assets to pay with so they can take advantage of market conditions anytime.  In addition, private users can request for payment from non-users, customers, friends, and family around the world and get paid in cryptocurrency while he/she receives the payment in local fiat currency regardless of the user's knowledge about cryptocurrency.

CryptoBank Africa provides a lot of benefits when making payments with your digital currency than the normal traditional payment methods in terms of transparency, fast, High fees involve due to middlemen e.t.c. nevertheless, The CryptoBank DApp(BANKER) allows users to store, receive and send their digital currency like Bitcoin, Ethereum, CBT, Nul, Ripple, Bitcoin Cash, Bitcoin SV, Litecoin, Binance Coin, Monero, Tron, Ethereum Classic, Dash, NEM, ZCash, Doge, Decred, Bitcoin Gold, Algorand, MonaCoin, Horizen, DigiByte, Komodo, Verge, ZCoin, Aion, Energi, Stratis, Syscoin, PIVX, Vertcoin, Groestlcoin, Einsteinium, NavCoin, SmartCash, Peercoin, LBRY Credits, Viacoin, Smileycoin e.t.c. CryptoBank DApp(BANKER) also allows users to interact with other useful crypto Apps/DApp like wallets, DEX, and Coin-market lists. To provide more advantages to our community/token holders, we integrated the CryptoBank wallet to the DApp(BANKER) which can be used to store, exchange, deposit, withdraw and make payments using your preferred digital currencies and their advantages when making a payment with CryptoBank native token ($CBT), users can easily make payments to retails with 0% fees in a matter of seconds regardless of the retail being crypto crackerjack.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
please provide an explanation how this is filecoin related or beneficial for the filecoin ecosystem

otherwise this is considered spam!

**Reply by cryptobanks-app** [reply]
I have replied to your mail...Thanks

**Comment by cryptobanks-app** [comment]
Hello from CryptoBank Africa

I'm sorry for the late reply, we are currently in a workshop...

CryptoBank Africa tends to integrate IPFS in our marketplace to enable
merchants/buyers to upload their goods/files in a decentralised
manner(IPFS).
Goods/files could be NFTs or other digital products that can be stored
online

Secondly, in other to minimize the costs of using the Blockchain and also
increasing security in our system, We plan to store data on-chain using
IPFS...

Thanks...
